### DISCUSSION TITLE
Allowlist / whitelist domain for all websites

### DISCUSSION THREAD

**Discussion by ThatIsAPseudo** [uBlockOrigin/uBlock-issues#2160] [Q&A] [answered]
Is there a way to have allowlists in order to allow connections to specific hosts even if they're in a blocklist, in case I want to override rules in some popular blocklist ?
For now I must either manually whitelist a host on every website, or I'd have to recompile a blocklist without the hosts I want to allow.
I know that a syntax exist in order to allow domains on a particular website [1] but it's not what I'm looking for, I don't want to have to specify websites.

[1] https://github.com/easylist/easylist/blob/master/easylist/easylist_allowlist.txt

**Comment by gorhill** [comment, marked as answer]
`domain=` is a filter option, you don't have to use it, in which case the exception will apply everywhere.
